model/src/main/java/jetbrains/jetpad/model/transform/Transformers.java

```diff
@@ -826,36 +826,40 @@ protected void doRemove() {
   }
 
   /**
-   * Select only those with the highest priority. Warning: target collection
-   * is not protected from outside writes.
-   * @param getPriority The greater is number, the higher is priority.
+   * Select only those with the highest priority. Null items are not allowed.
+   * Warning: target collection is not protected from outside writes.
+   * @param getPriority The greater is number, the higher is priority. Null priority
+   * is not allowed.
```

Comment:
What actually "is not allowed" means? Will it lead to an exception, or transfromer will just be in incorrect state? If first, what exception will be thrown? 
Such information is a part of transfromer contract, so some tests should be provider for cases above.
Also, there should be a comment that getPriority should always return the same value for same object.